core/network/impl/protocols/block_announce_protocol.cpp

```diff
@@ -369,12 +366,8 @@ namespace kagome::network {
 
           self->observer_->onBlockAnnounce(peer_id, block_announce);
 
-          auto hash = self->hasher_->blake2b_256(
-              scale::encode(block_announce.header).value());
-
-          self->peer_manager_->updatePeerStatus(
-              stream->remotePeerId().value(),
-              BlockInfo(block_announce.header.number, hash));
+          self->peer_manager_->updatePeerState(stream->remotePeerId().value(),
```

Comment:
It is needless in runtime, because streams of this and others protocols are creates after Identity protocol was done correctly. Any way, I'll add assert for that.